README.md

```
@@ -1,221 +1,167 @@
 
-```
-@article{lu2024aiscientist,
-  title={The {AI} {S}cientist: Towards Fully Automated Open-Ended Scientific Discovery},
-  author={Lu, Chris and Lu, Cong and Lange, Robert Tjarko and Foerster, Jakob and Clune, Jeff and Ha, David},
-  journal={arXiv preprint arXiv:2408.06292},
-  year={2024}
-}
-```
+#### 通过 Docker 运行程序
+
+然后通过 `docker compose` 来运行该项目： 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)